src/hooks/useMultiplayerState.ts

```diff
@@ -61,76 +61,76 @@ export function useMultiplayerState(roomId: string) {
     undoManager.redo();
   }, []);
 
-  /**
-   * Callback to update user's (self) presence
-   */
   const onChangePresence = useCallback((app: TldrawApp, user: TDUser) => {
-    if (!app.room) return;
-    room.setPresence({ id: app.room.userId, tdUser: user });
+    if (app.room) {
+      room.setPresence({ id: app.room.userId, tdUser: user });
+    }
   }, []);
 
-  /**
-   * Update app users whenever there is a change in the room users
-   */
   useEffect(() => {
-    if (!app || !room) return;
-
-    const unsubOthers = room.subscribe("others", (users) => {
-      if (!app.room) return;
-
-      const ids = users
-        .filter((user) => user.presence && user.presence.tdUser)
-        .map((user) => user.presence!.tdUser!.id);
-
-      // remove any user that is not connected in the room
-      Object.values(app.room.users).forEach((user) => {
-        if (user && !ids.includes(user.id) && user.id !== app.room?.userId) {
-          app.removeUser(user.id);
+    if (app && room) {
```

Comment:
I prefer the older version, in my opinion it's better to return from the function as early as possible, otherwise you end up with just some big nested block which makes the code harder to read.